src/senaite/core/upgrade/v02_05_000.py

```diff
@@ -293,3 +295,43 @@ def _remove_action(type_info, action_id):
     index = actions.index(action_id)
     type_info.deleteActions([index])
     return _remove_action(type_info, action_id)
+
+
+def import_workflow(tool):
+    """Import workflow step from profiles
+    """
+    logger.info("Import Workflow ...")
+    portal = tool.aq_inner.aq_parent
+    _run_import_step(portal, "workflow", profile=profile)
+    logger.info("Import Workflow [DONE]")
+
+
+def update_workflow_mappings_analyses(tool):
+    """Update the WF mappings for Analyses
+    """
+    logger.info("Updating role mappings for Analyses ...")
+    wf_id = "senaite_analysis_workflow"
+    query = {"portal_type": "Analysis"}
+    brains = api.search(query, ANALYSIS_CATALOG)
+    _update_workflow_mappings_for(wf_id, brains)
+    logger.info("Updating role mappings for Analyses [DONE]")
+
+
+def _update_workflow_mappings_for(wf_id, brains):
+    """Helper to update role mappings for the given brains
+    """
+    wf_tool = api.get_tool("portal_workflow")
+    workflow = wf_tool.getWorkflowById(wf_id)
+    total = len(brains)
+    for num, brain in enumerate(brains):
+        if num and num % 100 == 0:
+            logger.info("Updating role mappings: {0}/{1}".format(num, total))
+        if num and num % 1000 == 0:
+            logger.info("Comitting {0}/{1}".format(num, total))
```

Comment:
Typo: "comiting" -> "committing"

Reply:
I'm always doing this wrong 🙊
Fixed in 6c1b69afa